tests/testdata/buf.build/grpc-ecosystem/openapiv2/v2.15.0/petapis/plugin.sum

```diff
@@ -0,0 +1 @@
+h1:kbnot+zVGjxK+SAIaEjtKgvlHNA+4RVmP6bjvSYh9JA=
```

Comment:
Even though we don't have HTTP annotations in the input files, I verified that there is swagger generated for the services and types:
```
$ tree tests/testdata/buf.build/grpc-ecosystem/openapiv2/v2.15.0/*/gen
tests/testdata/buf.build/grpc-ecosystem/openapiv2/v2.15.0/eliza/gen
└── buf
    └── connect
        └── demo
            └── eliza
                └── v1
                    └── eliza.swagger.json
tests/testdata/buf.build/grpc-ecosystem/openapiv2/v2.15.0/petapis/gen
└── pet
    └── v1
        └── pet.swagger.json
```

We may want to add a more comprehensive test that has the http annotations in a follow up (especially for grpc-gateway which I think doesn't emit anything if they aren't defined).

Reply:
The gateway can be coerced to generate mappings even in the absence of annotations: https://github.com/grpc-ecosystem/grpc-gateway/blob/main/protoc-gen-grpc-gateway/main.go#L37

Reply:
Great! I'll look to set that option for the grpc-gateway test execution then.